test/distrib/bazel/python/WORKSPACE

```diff
@@ -20,3 +20,20 @@ local_repository(
     name = "some_other_repo",
     path = "../python_second_test_repo",
 )
+
+load("@upb//bazel:system_python.bzl", "system_python")
+
+system_python(
+    name = "system_python",
+    minimum_python_version = "3.7",
+)
+
+load("@system_python//:pip.bzl", "pip_parse")
+
+pip_parse(
```

Comment:
Are these dependencies actually necessary? I don't see why they should be. This looks like it's just pulling in numpy as a test dependency. This is also a breaking change to our downstream Python Bazel build consumers.

Reply:
Discussed offline. Will allow this for this PR, but this must be a blocker for the next gRPC release.